suite-native/intl/src/en.ts

```diff
@@ -757,29 +757,39 @@ export const en = {
                 defaultPassphrase: 'Passphrase wallet #{index}',
             },
         },
-        deviceChecks: {
-            title: 'Device checks',
-            firmwareAuthenticityCheck: {
-                title: 'Turn off firmware authenticity check',
-                subtitle:
-                    'Firmware authenticity check is a crucial security feature. We strongly recommend keeping it turned on.',
+        advanced: {
+            title: 'Advanced',
+            goodAdvice: 'We strongly recommend leaving these checks turned on.',
+            authenticityChecks: {
                 buttonTurnOff: 'Turn off',
                 buttonTurnOn: 'Turn on',
                 buttonLearnMore: 'Learn more',
-
-                turnOffModal: {
-                    title: 'Turn off firmware authenticity check',
-                    content:
-                        'Trezor Support will never ask you to turn off the firmware revision check. This feature is designed to protect your security.',
-                    item1: 'Only if the device has passed the check before',
+                toastOn: 'Check turned on',
+                toastOff: 'Check turned off',
+                turnOff: {
+                    content: 'This feature is designed to protect your security.',
```

Comment:
Using _protect your security_ without an object sounds weird to me. Is this really linguistically correct?

Reply:
"your security" is the object, or not?

Reply:
It doesn't sound right to me. I'd expect something like:
_"This feature is designed to protect your funds."_

Or:
"This feature is designed to increase your device security."

Definitely not a blocker, feel free to resolve.